src/OutboundServer/handlers.js

```diff
@@ -140,19 +164,47 @@ const healthCheck = async (ctx) => {
     ctx.response.body = '';
 };
 
+const metrics = async (ctx) => {
+    ctx.response.status = 200;
+    ctx.response.body = Metrics.getMetricsForPrometheus();
+};
+
 module.exports = {
     map: {
         '/': {
-            get: healthCheck
+            get: {
+                handler: healthCheck,
+                id: 'outbound_health_check',
+                enableSpan: false
```

Comment:
Same comment here applies to the previous InboundAPI comments.

Reply:
decided not to go with this option as name in route map requires the path to access